common-ui/tslint.json

```diff
@@ -25,7 +25,7 @@
     "max-line-length": [
       true,
       {
-        "limit": 100,
+        "limit": 140,
```

Comment:
Was this change intentional?

Reply:
yeah. We had discussed in slack before and most people seemed on board with increasing the limit.

I asked around the office real quick and it seems like opinions haven't changed.